docs/config.md

```diff
@@ -179,9 +179,9 @@ interface
 #### FSI
 
 * For `readData(Displacement)` or `DisplacementDelta`, you need the following:
-  * `type movingWallVelocity` for the interface (e.g. `flap`) in `0/U`,
-  * `type fixedValue` for the interface (e.g. `flap`) in the `0/pointDisplacement`, and
-  * `solver displacementLaplacian` in the `constant/dynamicMeshDict`. The solver [`RBFMeshMotionSolver` from solids4foam is also known to work](https://github.com/precice/openfoam-adapter/pull/241), currently (August 2022) with the develop branch of the OpenFOAM adapter and the nextRelease branch of solids4foam.
+  * `type movingWallVelocity` for the interface (e.g., `flap`) in `0/U`,
+  * `type fixedValue` for the interface (e.g., `flap`) in the `0/pointDisplacement`, and
+  * `solver displacementLaplacian` in the `constant/dynamicMeshDict`. The solver [`RBFMeshMotionSolver` from solids4foam is also known to work](https://github.com/precice/openfoam-adapter/pull/241), since the OpenFOAM adapter v1.2.0 and the nextRelease branch of solids4foam.
```

Comment:
Which one is 'the next one'. This sentence is already doomed to be outdated.

Reply:
This is how the branch is called, unfortunately:

```suggestion
  * `solver displacementLaplacian` in the `constant/dynamicMeshDict`. The solver [`RBFMeshMotionSolver` from solids4foam is also known to work](https://github.com/precice/openfoam-adapter/pull/241), since the OpenFOAM adapter v1.2.0 and the [`nextRelease` branch (v2) of solids4foam](https://github.com/solids4foam/solids4foam/commit/971d4c08f0171cfd8d53a6f831ce0745d98521de).
```